docs/rule-updates.mdx

```diff
@@ -12,6 +12,51 @@ import MoreHelp from "/src/components/MoreHelp"
 
 Welcome to monthly rule updates! This document includes selected new rules, removed false positives (FP), and other rule updates. These new rules and their updates are made by the Semgrep community and r2c.
 
+## October 2022
+
+### Community tier
+
+#### New rules from Semgrep community and r2c
+
+- Thanks @mtausig! <LinkToRegistryRule ruleId="csharp.dotnet.security.use_weak_rsa_encryption_padding" />
```

Comment:
@adamkvitek Can we link to the GitHub or other profiles for these contributors?

Reply:
Yes, I will try to find them!